samples/core/blur/blur.cpp

```diff
@@ -503,6 +510,11 @@ std::tuple<bool, bool, bool> BlurCppExample::query_capabilities()
                            use_subgroup_exchange_relative);
 }
 
+bool BlurCppExample::query_opencl_2_0_support()
+{
+    return cl::util::opencl_c_version_contains(device, "2.0");
```

Comment:
Nitpick: this helper function seems to be querying whether OpenCL **C** 2.0 is supported, not OpenCL 2.0.

Edit: Is checking for the OpenCL C version the right thing to do here?  Currently the C++ and C samples differ in what they are checking for.